scripts/build-local.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Please add a short comment here, at least with the same info as in the commit message.

Reply:
I renamed script to deploy-to-minikube.sh and added little comment.